static/js/util/editEducation.js

```diff
@@ -41,9 +41,9 @@ export function openNewEducationForm(level: string, index: number) {
 }
 
 export function deleteEducationEntry () {
-  const { saveProfile, profile, ui, deletionIndex } = this.props;
+  const { saveProfile, profile, ui } = this.props;
```

Comment:
i need `ui` in line 47 though. does that syntax make `ui` available as well? i thought it didn't